src/components/Footer/Footer.js

```diff
@@ -1,5 +1,93 @@
+import React from 'react';
+import styled from 'styled-components';
+import {
+  SERVICE_LIST,
+  QUESTIONS_LIST,
+  COMPANY_LIST,
+  CUSTOMER_SERVICE_LIST,
+  FOOTER_TEXT,
+} from './FooterData';
+
 function Footer() {
-  return <h2>Footer</h2>;
+  return (
+    <FooterWrap>
+      <FooterContent>
+        <FooterTop>
+          <ContentWrap>
+            <ContentTitle>서비스</ContentTitle>
+            {SERVICE_LIST.map(({ id, content }) => (
+              <ContentList key={id}>{content}</ContentList>
+            ))}
+          </ContentWrap>
+          <ContentWrap>
+            <ContentTitle>기타 문의</ContentTitle>
+            {QUESTIONS_LIST.map(({ id, content }) => (
+              <ContentList key={id}>{content}</ContentList>
+            ))}
+          </ContentWrap>
+          <ContentWrap>
+            <ContentTitle>회사</ContentTitle>
+            {COMPANY_LIST.map(({ id, content }) => (
+              <ContentList key={id}>{content}</ContentList>
+            ))}
+          </ContentWrap>
+          <ContentWrap>
+            <ContentTitle>고객센터</ContentTitle>
+            {CUSTOMER_SERVICE_LIST.map(({ id, content }) => (
+              <ContentList key={id}>{content}</ContentList>
+            ))}
+          </ContentWrap>
```

Comment:
```suggestion
          {data.map((ele) => {
             <ContentWrap>
                <ContentTitle>서비스</ContentTitle>
                {ele.map(({ id, content }) => (
                  <ContentList key={id}>{content}</ContentList>
                ))}
            </ContentWrap>
          })}
```
현재는 데이터가 다 분리되어 있어서 data.map이 총 네 번 사용되고 있는데,
1. 4개의 데이터를 하나의 배열안으로 묶는다.
2. 이중으로 map을 돌리는 방법을 한 번 고민해보세요!

Reply:
이중 mapping 처리했습니다! 안될 줄 알았는데 되니까 뿌듯하네요...핫